drivers/fuel_gauge/fuel_gauge_syscall_handlers.c

```diff
@@ -71,9 +73,6 @@ static inline int z_vrfy_fuel_gauge_set_props(const struct device *dev, fuel_gau
 
 	int ret = z_impl_fuel_gauge_set_props(dev, k_props, k_vals, len);
 
-	/* We only copy back vals because props will never be modified */
-	K_OOPS(k_usermode_to_copy(vals, k_vals, len * sizeof(union fuel_gauge_prop_val)));
```

Comment:
Good catch. This is a hold-over from when the API used to set individual return codes on the first invocation of a batch property set call.